src/routes/AppContent/index.tsx

```diff
@@ -81,22 +84,24 @@ const AppContent = () => {
                 {isLoadingActiveAccount && !isEndpointRoute ? (
                     <Loader />
                 ) : (
-                    <div className='app-content__body'>
-                        <Tabs
-                            activeTab={activeTab}
-                            className='app-content__tabs'
-                            onChange={index => {
-                                setActiveTab(tabRoutesConfiguration[index].name);
-                                history.push(tabRoutesConfiguration[index].path);
-                            }}
-                            variant='secondary'
-                        >
-                            {tabRoutesConfiguration.map(route => (
-                                <Tab key={route.name} title={route.name!} />
-                            ))}
-                        </Tabs>
-                        <Router />
-                    </div>
+                    <>
+                        <div className='app-content__body'>
+                            <Tabs
+                                activeTab={activeTab}
+                                className='app-content__tabs'
+                                onChange={index => {
+                                    setActiveTab(tabRoutesConfiguration[index].name);
+                                    history.push(tabRoutesConfiguration[index].path);
+                                }}
+                                variant='secondary'
+                            >
+                                {tabRoutesConfiguration.map(route => (
+                                    <Tab key={route.name} title={route.name!} />
```

Comment:
```suggestion
                                    <Tab key={route.name} title={route.name} />
```
why the need for the none-null coercion operator here?

Reply:
weird, dont think we need it actually, ill remove it